plasmapy/particles/ionization_state.py

```diff
@@ -767,6 +839,10 @@ def _get_states_info(self, minimum_ionic_fraction=0.01) -> List[str]:
                     value = "{:.2e}".format(state.number_density.si.value)
                     state_info += f"    n_i = {value} m**-3"
 
+                if np.isfinite(state.T_i):
+                    value = "{:.2e}".format(state.T_i.si.value)
```

Comment:
I keep forgetting that there are other ways to format strings besides f-strings. 😬

Reply:
... it shows that I copy pasted the block above, doesn't it? :D